src/styles/main.scss

```
@@ -1,3 +1,55 @@
+@import './variables';
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  box-sizing: border-box ;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left;
+  width: $calendar-width;
+  padding: 10px;
+  gap: $gap;
+
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    width: $box-size;
+    height: $box-size;
+    position: relative;
+    font-size: 30px;
+    align-items: center;
+    justify-content: center;
+    background-color: $color-of-background;
+    border: 1px solid $color-of-border;
+    transition: 0.5s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition:  transform $transitionDuration;
```